spec/helpers/view_helper_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ViewHelper do
+  include ViewHelper
```

Comment:
```suggestion
describe ViewHelper, type => :helper do
```

We shouldn't really need to include it...

> Helper specs are marked by :type => :helper or if you have set
config.infer_spec_type_from_file_location! by placing them in spec/helpers.

Reply:
updated